examples/largeclusters/swarmmode-vmas.json

```diff
@@ -0,0 +1,97 @@
+{
+  "apiVersion": "vlabs",

```

Comment:
Synced on email. I have test both Swarm Mode VMSS and VMAS large cluster scenarios with agent pool count of 1 in all pools.